TodoList.Web/TodoList.Web.csproj

```diff
@@ -34,6 +34,7 @@
 
 
   <ItemGroup>
+    
```

Comment:
It seems like we don't need this blank line here.

Reply:
Done.